src/postgres/tree/PostgresServerTreeItem.ts

```diff
@@ -145,3 +159,16 @@ export class PostgresServerTreeItem extends AzureParentTreeItem<ISubscriptionCon
         return !!version && gte(version, '11.0.0');
     }
 }
+
+async function validateDatabaseName(name: string, getChildrenTask: Promise<AzExtTreeItem[]>): Promise<string | undefined | null> {
+    if (!name || name.length < 1) {
+        return localize('NameCannotBeEmpty', 'Name cannot be empty.');
+    }
+    const currDatabaseList = await getChildrenTask;
+    const currDatabaseNames: string[] = [];
+    currDatabaseList.map(database => currDatabaseNames.push(database.label));
+    if (currDatabaseNames.includes(name)) {
```

Comment:
Are database names case sensitive? If not, you should do `toLowerCase` on both `currDatabaseNames` and `name` to prevent the same name with different casing

Reply:
They are case sensitive. Actually, except for "Name cannot be empty", there aren't any other restrictions to database names. Only pgAdmin has one extra check to see if a database name already exists under a server. We can still create duplicate database names from the extension if we allow. Should we follow pgAdmin and restrict duplicates?